lib/tools/android-manifest.js

```diff
@@ -101,7 +101,11 @@ manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk)
     log.debug(`badging act: ${apkActivity}`);
     return {apkPackage, apkActivity};
   } catch (e) {
-    log.errorAndThrow(`packageAndLaunchActivityFromManifest failed. Original error: ${e.message}`);
+    let stdErrInfo = "";
+    if (e.stderr) {
+      stdErrInfo = `\nStd err: ${e.stderr}`;
+    }
+    log.errorAndThrow(`packageAndLaunchActivityFromManifest failed. Original error: ${e.message}` + stdErrInfo);
```

Comment:
just change it it to

```
log.errorAndThrow(`packageAndLaunchActivityFromManifest failed. Original error: ${e.message}` +
                                 (e.stderr ? `; StdErr: ${e.stderr}` : ''));
```
and all the code above is not needed anymore

Reply:
I have fixed.